Signalized Intersection Manager support

MishkaMN opened the pull request:
<!-- Thanks for the contribution, this is awesome. -->

# PR Details
## Description

<!--- Describe your changes in detail -->
This PR supports the PR here https://github.com/usdot-fhwa-stol/carma-platform/pull/1547
CarmaTrafficSignal now has exit lanelets describing which lanes the signal controls in an intersection.

## Related Issue
https://github.com/usdot-fhwa-stol/carma-platform/issues/1548
<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

## Motivation and Context
This change is to support multiple directions from a lane
<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?
Local integration tested
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Defect fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [X] I have added any new packages to the sonar-scanner.properties file
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

MishkaMN committed:
combined

msmcconnell commented:
This PR could use more description

msmcconnell reviewed (changes requested):
Looks good, but I believe this change also changes the XML description so please update that in the README accordingly.

MishkaMN committed:
Update RegulatoryElements.md

MishkaMN commented:
> Looks good, but I believe this change also changes the XML description so please update that in the README accordingly.

Just updated the README.md and updated the PR description

MishkaMN committed:
multuple stoplines

MishkaMN committed:
Merge branch 'feature/signalized_intersection' of https://github.com/usdot-fhwa-stol/autoware.ai into feature/signalized_intersection
